### ISSUE TITLE
OpenLab HomePage: Missing page landmarks

### ISSUE THREAD

**Issue by beckej13820** [cuny-academic-commons/commons-in-a-box#384]
I've been looking at the OpenLab theme with an accessibility checker called [Axe Dev Tools](https://www.deque.com/axe/devtools/).

The OpenLab Homepage on my site has 121 instances of the error "All page content should be contained by landmarks."

When I inspect the HTML of the page, the areas of the page are identified in IDs, but they don't have the corresponding role associated with the IDs. It appears that the main problem is that there is no "role=main" or <main> element on the page. You can either use the HTML element or if you are using <div>s you can you use the ARIA label role to mark it as a landmark. All content is supposed to be inside landmarks

There is a div with the id="openlab-main-content" and I think if you just added role=main to that it would remove most of the issues, but there might be other areas that also need to be done such as Header, Nav, Main, Footer, or Aside to get all content inside a landmark.

Axe labels this as an accessibility best practice, but not a requirement. It helps screenreaders but isn't part of a WCAG Success Criteria.

**Comment by boonebgorges** [comment]
Thanks for the report. It's a good suggestion. I've looked through the theme's primary top-level templates and added 'main' and 'complementary' roles as appropriate.
